framework/src/org/checkerframework/framework/type/TypeFromExpressionVisitor.java

```diff
@@ -362,14 +370,16 @@ public AnnotatedTypeMirror visitWildcard(WildcardTree node,
     @Override
     public AnnotatedTypeMirror visitPrimitiveType(PrimitiveTypeTree node,
                                                   AnnotatedTypeFactory f) {
-        // for e.g. "int.class"
+        // Node is the primitive type when primitive class literals are visited.
```

Comment:
>Can this ever be reached?

Yes, Dataflow calls visits the "int" node separately and calls CFAbstractTransfer.getValueFromFactory on this node. CFAbstractTransfer.getValueFromFactory call getAnnotatedType which (indirectly) calls this method.